public/examples/upload/uploadApp.js

```diff
@@ -270,36 +270,21 @@
         }
 
         var uploadHttpConcurrentProgress;
-        var baseUrl;
+        var baseUrl = (version === 'IPv6') ? testPlan.baseUrlIPv6NoPort : testPlan.baseUrlIPv4NoPort;
+        for (var i = 0; i < ports.length; i++) {
+            for (var b = 0; b < 6; b++) {
+                urls.push('http://' + baseUrl + ':' + ports[i] + '/upload');
+            }
+        }
         //TODO needs to removed once we know the issues  with ie
         if (navigator.appVersion.indexOf("MSIE") != -1 || navigator.appVersion.indexOf("Trident") != -1 || navigator.appVersion.indexOf("Edge") != -1) {
             var isIE = true;
```

Comment:
we don't have to make the flag to false and make it true.. it enters the loop only if it is true